server/src/main/java/org/candlepin/model/Content.java

```diff
@@ -148,6 +172,103 @@ public Content(String id) {
         this.setId(id);
     }
 
+    /**
+     * Creates a shallow copy of the specified source content. Owners, attributes and content are
+     * not duplicated, but the joining objects are (ContentAttribute, ContentContent, etc.).
+     * <p/>
+     * Unlike the merge method, all properties from the source content are copied, including the
+     * state of any null collections and any identifier fields.
+     *
+     * @param source
+     *  The Content instance to copy
+     */
+    protected Content(Content source) {
+        this.setUuid(source.getUuid());
+        this.setId(source.getId());
+
+        this.setCreated(source.getCreated());
+        this.setUpdated(source.getUpdated());
+
+        this.setType(source.getType());
+        this.setLabel(source.getLabel());
+        this.setOwners(source.getOwners());
+        this.setName(source.getName());
+        this.setVendor(source.getVendor());
+        this.setContentUrl(source.getContentUrl());
+        this.setRequiredTags(source.getRequiredTags());
+        this.setReleaseVer(source.getReleaseVer());
+        this.setGpgUrl(source.getGpgUrl());
+        this.setMetadataExpire(source.getMetadataExpire());
+        this.setModifiedProductIds(source.getModifiedProductIds());
```

Comment:
Yeah. Looks like this is one of the collections that is modified and set externally in a number of places, so copying it is probably the way to go.